db/migrations/postgres/000093_notify_admin.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS NotifyAdmin (
+    Id varchar(26) NOT NULL,
+    UserId varchar(26) NOT NULL,
+    CreateAt bigint DEFAULT NULL,
+    RequiredPlan varchar(26) NOT NULL,
+    RequiredFeature varchar(26) NOT NULL,
+    Trial BOOLEAN NOT NULL,
+    PRIMARY KEY (Id)
```

Comment:
Yes this is awesome. I am wondering (1/5) if we can first hold off on this because maybe requirements could change and duplicate requests could be allowed in the future (we won't have to do an alter migration). Atm, other teams have it that the notification keeps going out as long as the user keeps clicking notify.